cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -297,6 +313,8 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 	cvc.Spec.Policy = volumePolicy.Spec
 	cvc.Status.Phase = apis.CStorVolumeClaimPhaseBound
 	cvc.Status.Capacity = cvc.Spec.Capacity
+	// update volume replica pool information on cvc spec and status
+	addReplicaPoolInfo(cvc, poolNames)
```

Comment:
how about changing it as receiver function like cvc.addReplicaPoolInfo(poolNames)